microsoft-edge/progressive-web-apps-chromium/experimental-features/index.md

```diff
@@ -0,0 +1,469 @@
+---
+description: The latest experimental features in Microsoft Edge for Web Apps
+title: Experimental features | Progressive Web Apps
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/16/2021 
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, experiment, progressive web apps, web apps, PWAs, PWA
+---
+# Experimental features in Progressive Web Apps
+
+Microsoft Edge provides access to experimental features that are still in development.  You may test and [provide feedback](#providing-feedback-on-experimental-features) on these features to help to determine when and if to release them.  
+
+While experimental features are available in all channels of Microsoft Edge, you will get the latest experimental features using the Microsoft Edge Canary channel.  
+
+## Turn on experimental features  
```

Comment:
This turns on the feature for the developer on their machine. Do we want to mention Origin Trials here for features that support them?  This way developers can have some of these features enabled for their users and gives feedback on the overall experience..